packages/core/src/types.ts

```diff
@@ -7,3 +7,19 @@ export type StoreDeploymentConfig = (
 ) => Promise<string>
 
 export type RelayProposal = (proposalRequest: ProposalRequest) => Promise<void>
+
+export type SphinxLock = {
+  format: string
+  projects: {
+    [projectName: string]: {
+      projectId: string
+      projectName: string
+      defaultSafe: {
+        safeName: string
+        owners: string[]
+        threshold: string
+        saltNonce: string
+      }
+    }
+  }
+}
```

Comment:
This is the format of the `sphinx.lock` file, a couple notes on it:
1. `projectId` is the literal id stored in the DB for this project, it is generated by us and not by the user. We don't currently use it for anything, but I think it could be useful. 
2. I put all of the Gnosis Safe information into a `defaultSafe` object instead of just having top-level fields because I want this type to accommodate future features like importing external safes.